### DISCUSSION TITLE
API Details and Routes for E-Commerce application

### DISCUSSION THREAD

**Discussion by GouthamGunji** [akash-coded/spring-framework#63] [Q&A] [answered]
Hi @akash-coded , Could you please share the API details and routes for the same - that we have discussed for E-Commerce Application here.

**Comment by akash-coded** [comment, marked as answer]
Here you go. Sharing the whiteboard here.

![sportyshoes-ecommerce-api-project-description](https://user-images.githubusercontent.com/60097560/167265490-b5e86625-d884-4f73-b497-1788dd15d3c3.png)
